src/ui/pages/codi/zcl_abapgit_gui_page_whereused.clas.abap

```diff
@@ -0,0 +1,222 @@
+
+  ENDMETHOD.
+
+
+  METHOD init_table_component.
+
+    DATA ls_sorting_state TYPE zif_abapgit_html_table=>ty_sorting_state.
+
+    IF mi_table IS BOUND.
+      RETURN.
+    ENDIF.
+
+    IF ls_sorting_state-column_id IS INITIAL.
+      ls_sorting_state-column_id = 'package'.
+    ENDIF.
+
+    mi_table = zcl_abapgit_html_table=>create(
+      ii_renderer              = me
+      is_initial_sorting_state = ls_sorting_state ).
+    mi_table->define_column_group(
+        iv_group_title = 'Repo object'
+        iv_group_id    = '' " No need
+      )->define_column(
+        iv_column_id    = 'dep_package'
+        iv_column_title = 'Pkg'
+      )->define_column(
+        iv_column_id    = 'dep_obj_type'
+        iv_column_title = 'Type'
+      )->define_column(
+        iv_column_id    = 'dep_obj_name'
+        iv_column_title = 'Obj name'
+      )->define_column(
+        iv_column_id    = 'dep_used_obj'
+        iv_column_title = 'Used obj' ).
+    mi_table->define_column_group(
+        iv_group_title = 'Used in'
+        iv_group_id    = 'where' " Needed for CSS
+      )->define_column(
+        iv_column_id    = 'package'
+        iv_column_title = 'Pkg'
+      )->define_column(
+        iv_column_id    = 'obj_type'
+        iv_column_title = 'Type'
+      )->define_column(
+        iv_column_id    = 'obj_name'
+        iv_column_title = 'Obj name' ).
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_gui_event_handler~on_event.
+
+    IF mi_table->process_sorting_request( ii_event->mv_action ) = abap_true.
+      rs_handled-state = zcl_abapgit_gui=>c_event_state-re_render.
+      RETURN.
+    ENDIF.
+
+    CASE ii_event->mv_action.
+      WHEN c_action-refresh.
+        rs_handled-state = zcl_abapgit_gui=>c_event_state-re_render.
+      WHEN OTHERS.
+        RETURN.
+    ENDCASE.
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_gui_hotkeys~get_hotkey_actions.
+
+    DATA ls_hotkey_action LIKE LINE OF rt_hotkey_actions.
+
+    ls_hotkey_action-ui_component = c_title.
+
+    ls_hotkey_action-description = |Refresh|.
+    ls_hotkey_action-action = c_action-refresh.
+    ls_hotkey_action-hotkey = |r|.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_gui_menu_provider~get_menu.
+
+    ro_toolbar = zcl_abapgit_html_toolbar=>create( )->add(
+      iv_txt = 'Refresh'
+      iv_act = c_action-refresh ).
+
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_gui_page_title~get_page_title.
+    rv_title = c_title.
+  ENDMETHOD.
+
+
+  METHOD zif_abapgit_gui_renderable~render.
+
+    DATA lt_where_used TYPE zcl_abapgit_where_used_tools=>ty_dependency_tt.
+
+    register_handlers( ).
+    init_table_component( ).
+
+    CREATE OBJECT ri_html TYPE zcl_abapgit_html.
+
+    ri_html->div(
+      iv_class   = 'wu-header'
+      iv_content = |Where used for package {
+        zcl_abapgit_gui_chunk_lib=>render_package_name( mv_package )->render( iv_no_line_breaks = abap_true )
+        } and it's subpackages| ).
```

Comment:
Depending on repo setting:

"Where-used of objects in package XYZ in other packages" 
or 
"Where-used of objects in package XYZ and its subpackages in other packages"